releasenotes/notes/cx_cz_synthesis-3d5ec98372ce1608.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Add a new synthesis algorithm `.synth_cx_cz_depth_line_my` of a CX circuit 
```

Comment:
Could we add an example here? Including both how we should run the code and what the output would look like?